app/views/components/gds/tariffLink.scala.html

```diff
@@ -18,4 +18,4 @@
 
 @(linkText: String, linkUrl: String)(implicit messages: Messages, appConfig: AppConfig)
 
-<a class="govuk-link govuk-link--no-visited-state" target="_blank" href="@linkUrl">@linkText</a>
+<a class="govuk-link govuk-link--no-visited-state" target="_blank" href="@linkUrl">@linkText</a>
```

Comment:
This empty line has been removed on purpose as it caused a space between the anchor link and the terminating full stop (defined in the messages file).